azavea/noaa-hydro-data
# Add Standard Scripts

## Overview

Add Azavea standard scripts to make development easier.

Update the README to reflect simpler development steps.

Make the dev environment more general and installable on any machine rather than a specific one.

## Testing Instructions

- Check out this branch and run `./scripts/setup`
- Run `./scripts/server`
  - [x] Ensure you are given a Jupyter Notebook URL at the bottom of the output
  - [x] Ensure clicking it takes you to a working Jupyter install
  - [x] Ensure the Notebooks work as expected
- While `./scripts/server` is running, run `./scripts/psql`
  - [x] Ensure you are in the database and can interact as expected
